src/app/components/Home/ListOfAddresses/index.tsx

```diff
@@ -11,21 +11,21 @@ interface IarrCity {
   [key: number]: string;
 }
 interface IListOfAddresses {
-  addresesData: Array<Array<IarrCity>>;
+  addresesData: Array<IarrCity>;
 }
 
-const ListOfAddresses: React.FC<IListOfAddresses> = ({ addresesData }) => {  
+const ListOfAddresses: React.FC<IListOfAddresses> = ({ addresesData }) => {
   return (
     <>
       <NavListOfAddress>
-        {addresesData?.map(([address, office_id], index) => (            
+        {addresesData?.map((element: any, index) => (
           <ListItem key={`addres-${index}`}>
             <LocationOnIcon color="secondary" />
             <LinkOfAdress
-              to={`/office/${office_id}`}
-              id={address as string}
+              to={`/office/${element.id}`}
+              id={element.addres as string}
```

Comment:
It would be better to use id in as `key`  and `id` for each item.

Reply:
you mean it -
![image](https://user-images.githubusercontent.com/36860438/130738398-7b7ce26f-8370-4e09-9132-e1c573563359.png)